src/components/UserProfile.jsx

```diff
@@ -1,15 +1,51 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import '../styles/index.scss';
 import ninja from '../assets/images/ninja.jpg';
+import { increment, decrement } from '../actions';
 
 const UserName = () => <h2>Forsetti</h2>;
 
 const Avatar = () => <img src={ninja} alt="ninja"/>;
 
-const Userprofile = () => 
-    <div>
-        <UserName />
-        <Avatar />,
-    </div>
+class Userprofile  extends Component{
```

Comment:
@Chiazokam when making use of props in a component class, I think it is a good convention to always set up a constructor function with props passed to it, any reason why you did not use a constructor function in this case?

Reply:
@davidokonji Why do you think it is a good practice?

Reply:
I thought about this. But considered that since we are using redux it may not be necessary. I may be wrong. What do you think?

Reply:
@paradisekelechi according to react docs, when using component states it is good practice to add a constructor function with props as a parameter and a super() method called within the constructor function.